views/scss/qti/_choice.scss

```diff
@@ -40,6 +40,14 @@
         // ol
         .choice-area {
             margin-top: 25px;
+            .label-content {
+                display: inline-block;
+                .qti-block {
+                    img {
+                        width: 100%;
+                    }
+                }
+            }
```

Comment:
Yes, you're completely right. This does not work as intended. I'm going to close this pr. Perhaps there is a way to accomplish creating an x only for the length of the content, as well as setting the img width to a percentage of the item's full available space, but I'm not sure how to as of now.